src/bridge/jsHelpers.ts

```diff
@@ -119,20 +118,15 @@ Operation[] {
   return all;
 }
 
-export const mergeNfts = (
-  oldNfts: NFT[] | undefined,
-  newNfts: NFT[] | undefined
-): NFT[] => {
-  if (!newNfts?.length) return oldNfts ?? [];
```

Comment:
this should fix the fact sending the LAST nft will correctly update the nft list.